docs/configuration.md

```diff
@@ -1,6 +1,6 @@
 # Configuration
 
-Workflows are configured in a file called `.probot.js` in your repository.
+Workflows are configured in a file called `.github/.probot.js` in your repository.
```

Comment:
Since this is in `.github`, does the `probot.js` file need to start with a `.`?

Reply:
Not really. Do you prefer without? Both seems acceptable, so as you want.